docs/en_US/getting-started/getting-started.md

```diff
@@ -1,113 +1,247 @@
 
-    Full example please check [`emqxenterprise-full.yaml`](https://github.com/emqx/emqx-operator/blob/main/config/samples/emqx/v1beta3/emqxenterprise-full.yaml).
+For detailed explanation of each field, please refer to [v1beta3-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md)
 
-    Detailed explanation of each field please check [v1beta3-reference](https://github.com/emqx/emqx-operator/blob/main/docs/en_US/reference/v1beta3-reference.md)
+2. Check whether the EMQX cluster is ready
 
-2. Check EMQX Custom Resource status
+```bash
+kubectl get emqxenterprise emqx-ee -o json | jq ".status.emqxNodes"
+```
 
-    ```
-    $ kubectl get pods
-    $ kubectl get emqxenterprise emqx-ee -o json | jq ".status.emqxNodes"
-    ```
+The output is similar to:
+
+```bash
+[
+   {
+     "node": "emqx-ee@emqx-ee-0.emqx-ee-headless.default.svc.cluster.local",
+     "node_status": "Running",
+     "otp_release": "24.3.4.2/12.3.2.2",
+     "version": "4.4.14"
+   },
+   {
+     "node": "emqx-ee@emqx-ee-1.emqx-ee-headless.default.svc.cluster.local",
+     "node_status": "Running",
+     "otp_release": "24.3.4.2/12.3.2.2",
+     "version": "4.4.14"
+   },
+   {
+     "node": "emqx-ee@emqx-ee-2.emqx-ee-headless.default.svc.cluster.local",
+     "node_status": "Running",
+     "otp_release": "24.3.4.2/12.3.2.2",
+     "version": "4.4.14"
+   }
+]
+```
 
 ## Remark
-1. The `MixedProtocolLBService` feature is enabled by default in Kubernetes 1.24 and above. For its documentation, please refer to: [ MixedProtocolLBService ](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features). The `MixedProtocolLBService` attribute allows different protocols to be used within the same Service instance of type `LoadBalancer`. Therefore, if the user deploys the EMQX cluster on Kubernetes and uses the `LoadBalancer` type of Service, there are both TCP and UDP protocols in the Service, please pay attention to upgrading the Kubernetes version to 1.24 or above, otherwise the Service creation will fail.
\ No newline at end of file
+
+1. The `MixedProtocolLBService` feature is enabled by default in Kubernetes 1.24 and above. For its documentation, please refer to: [MixedProtocolLBService ](https://kubernetes.io/zh-cn/docs/reference/command-line-tools-reference/feature-gates /#feature-gates-for-alpha-or-beta-features). The `MixedProtocolLBService` attribute allows different protocols to be used within the same Service instance of type `LoadBalancer`. Therefore, if the user deploys the EMQX cluster on Kubernetes and uses the `LoadBalancer` type of Service, there are both TCP and UDP protocols in the Service, please pay attention to upgrading the Kubernetes version to 1.24 or above, otherwise the Service creation will fail.
```

Comment:
Move this block to `Prepare the environment`

Reply:
> Move this block to `Prepare the environment`

Done